index.html

```diff
@@ -30,6 +30,70 @@ <h1 class="first-screen__title">Мы создаем легкие решения
                     </div>                
                 </article>
         </section>
+        <section class="services">
+            <div class="services__container">
+                <header class="services__header">
```

Comment:
це не хедер, а звичайний контейнер для тайтла і текста під ним
```suggestion
                <div class="services__header">
```